lblod/frontend-form-builder
# SFB-143: show shacl validation report in console

## ID
[SFB-143](https://binnenland.atlassian.net/browse/SFB-143)

 ## Description

The SHACL validator returns a `Report` object./ Format this `Report` so it easy to read for a developer. Only validate the shapes when the use editing in the `code` tab.

 ## Type of change


 ## How to test

- Update the ttl code in the `code` tab and see if you get a result with a debounce of 500ms/

 ## Links to other PR's

 - https://github.com/lblod/frontend-form-builder/pull/107
